packages/jest-worker/src/Farm.ts

```diff
@@ -121,15 +115,11 @@ export default class Farm {
   }
 
   private _process(workerId: number): Farm {
-    if (this._isLocked(workerId)) {
-      return this;
-    }
+    if (this._isLocked(workerId)) return this;
```

Comment:
Is avoiding curly braces the Jest code style? We should adhere to whatever the project general preference is

Reply:
we don't lint for it, but generally prefer to have the curlies. Here it'd be nice to keep just to keep the noise down in the diff